### ISSUE TITLE
Does DALEC need exact dates?/met not updating properly

### ISSUE THREAD

**Issue by annethomas** [PecanProject/pecan#1565]
@mdietze I'm having a problem with met files not updating correctly when I run new additional years with DALEC, with the output only going as far as the last met file downloaded even though the run asked for more. The run doesn't fail; it just has empty output for the additional years (i.e. out.txt stops early and the .nc files are all NA). @tonygardella wanted to know if DALEC should be asking for exact dates in the registry file (like SIPNET) and if this would fix the problem.
Also, updating _does_ happen; in run 1001229036 the workflow.Rout shows additional downloads happening and the met file being updated, but then convert.input seems to revert to the old one.

The old end year was 2011 and the new end year is 2013; here's the tail end of the updating to the point where it reverts ("updated met path").
```
[1] 2013
2017-08-01 21:25:35 WARN   [PEcAn.DALEC::met2model.DALEC] : 
   Leaf N not specified, setting to default: 2.5 
2017-08-01 21:25:35 WARN   [PEcAn.DALEC::met2model.DALEC] : 
   total plant-soil hydraulic resistance (MPa.m2.s/mmol-1) not specified, 
   setting to default: 1 
2017-08-01 21:25:35 WARN   [PEcAn.DALEC::met2model.DALEC] : 
   maximum soil-leaf water potential difference (MPa) not specified, 
   setting to default: -0.8 
> print(remoteout)
                                                                                                                                  file
1 /fs/data1/pecan.data/dbfiles/Fluxnet2015_DALEC_site_1-5158//FLX_US-Wkg_FLUXNET2015_SUBSET_HH_2004-2014_1-3.2005-01-01.2013-12-31.dat
           host   mimetype        formatname           startdate
1 pecan2.bu.edu text/plain DALEC meteorology 2004-12-31 19:00:00
              enddate
1 2013-12-30 19:00:00
                                                               dbfile.name
1 FLX_US-Wkg_FLUXNET2015_SUBSET_HH_2004-2014_1-3.2005-01-01.2013-12-31.dat
> fp <- file('/fs/data1/pecan.data/dbfiles/Fluxnet2015_DALEC_site_1-5158//pecan-9d47b243380c1bb9b1f98ec8910ae6', 'w')
> ign <- serialize(remoteout, fp)
> close(fp)
> 
[1] 0
2017-08-01 21:25:35 INFO   [convert.input] : 
   RESULTS: Convert.Input 
2017-08-01 21:25:35 INFO   [convert.input] : 
   /fs/data1/pecan.data/dbfiles/Fluxnet2015_DALEC_site_1-5158//FLX_US-Wkg_FLUXNET2015_SUBSET_HH_2004-2014_1-3.2005-01-01.2013-12-31.dat 
   pecan2.bu.edu text/plain DALEC meteorology 1104537600 1388448000 
   FLX_US-Wkg_FLUXNET2015_SUBSET_HH_2004-2014_1-3.2005-01-01.2013-12-31.dat 
2017-08-01 21:25:35 INFO   [convert.input] : 
   file host mimetype formatname startdate enddate dbfile.name 
2017-08-01 21:25:35 INFO   [.met2model.module] : 
   Finished Model Specific Conversion 1000013495 
2017-08-01 21:25:35 WARN   [PEcAn.utils::logger.warn] : 
   Connection created outside of PEcAn.db package 
2017-08-01 21:25:35 DEBUG  [PEcAn.utils::logger.debug] : 
   updated met path: 
   /fs/data1/pecan.data/dbfiles/Fluxnet2015_DALEC_site_1-5158/FLX_US-Wkg_FLUXNET2015_SUBSET_HH_2004-2014_1-3.2005-01-01.2011-12-31.dat 
```

**Comment by mdietze** [comment]
@annethomas @tonygardella yes, it looks like DALEC requires exact dates in the met file -- start and end date don't appear to be passed in anywhere in the configs. If that's not in the registry then update that, remake, and then give it another try.

For future ref, it's easier to find things by workflow ID, since that's what's in history.php and the folder names.

**Comment by tonygardella** [comment]
Closed because #1567
